.eslintrc

```diff
@@ -2,6 +2,13 @@
   "extends": ["airbnb", "plugin:prettier/recommended"],
   "plugins": ["dependencies"],
   "parser": "babel-eslint",
+  "settings": {
+    "import/resolver": {
+      "webpack": {
+        "config": "webpack.config.js"
+      }
+    }
+  },
```

Comment:
What this piece of code is doing? Just curious

Reply:
The webpack configuration has been modified so that there are no long relative paths. But eslint doesn't see these settings and throws errors. This setting indicates the path to the webpack configuration where the rules for importing files are.